src/activity-functions-function-notation.xml

```diff
@@ -637,6 +549,10 @@
               </table>
               <p><var name="$table2" form="buttons" /></p>
             </statement>
+            <solution>
+              <p>Table A is not a function because the input <m>x = 5</m> yeilds more than one result.</p>
+              <p>Table B is a function because each input yeilds only one result. It does not matter that the results are all the same. This is an example of a <term>constant function</term>.</p>
```

Comment:
Noticed a spelling typo with "yeilds". One helpful thing about making the print version is that it is relatively easy to run a spell check on the .tex file and find this type of spelling error. (It's still hard to find errors like "he" instead of "the" though.)